packages/base/Dropdown/index.js

```diff
@@ -147,6 +155,28 @@ const Dropdown = ({ input, ...rest }) => {
                 setAsyncOptions(options);
               }
 
+              // check for subtext and display if exists
+              const checkSubText = options.some(option => {
+                return 'subtext' in option;
+              });
+              if (checkSubText) {
+                options = options.map(option => {
+                  return {
+                    label:
+                      typeof option.subtext !== 'undefined' ? (
+                        <span>
+                          {option.label} <br />
+                          <Box color="gray5">{option.subtext}</Box>
```

Comment:
```suggestion
                          <Box color="grays.4">{option.subtext}</Box>
```

We should use the updated array syntax for grays. [We are trying to remove these](https://github.com/spraoi/barrel-ui/blob/master/src/themes/default.js#L72-L83)